acceptance/setup/aio/pre-suite/021_InstallAristaModule.rb

```diff
@@ -12,6 +12,8 @@
 
   step 'add puppet user to switch' do
     switchs.each do |switch|
+      on(switch, "useradd -U puppet")
```

Comment:
BTW, the line "useradd -U puppet" adds the puppet user and group (the -U flag adds group)